Preparation/TestProject1/MyStackInterTest.cs

```diff
@@ -0,0 +1,82 @@
+using MyStack;
+using System;
+using System.Data;
+using System.Linq;
+using Xunit;
+
+namespace Preperation.Tests
+{
+    public class MyStackInterTest
+    {
+        [Fact]
+        public void Push_AddsOneElementToTheCollection()
+        {
+            MyStack<int> myStack = new MyStack<int>();
+
+            myStack.Push(1);
+            myStack.Push(2);
+            myStack.Push(3);
+            myStack.Push(4);
+
+            Assert.Equal(4, myStack.Count());
+            Assert.Equal(4, myStack.Last());
+        }
+        [Fact]
+        public void Peek_ReturnsLastPushedElementsInTheCollectionTest()
+        {
+            MyStack<int> myStack = new MyStack<int>();
+            myStack.Push(1);
+            myStack.Push(20);
+            myStack.Push(30);
+
+            int result = myStack.Peek();
+
+            Assert.Equal(30, result);
+            Assert.Equal(3, myStack.Count());
```

Comment:
This should be a separate test that specifically checks that stack elements count is not changed during peek. Similar should be applied for checking Pop work

Reply:
So I need to write more to test-method, which check that count is correct work in Peek() and Pop(), right?

Reply:
Instead of this one `Peek_ReturnsLastPushedElementsInTheCollectionTest` you should have two
`Peek_ReturnsLastPushedElement` and `Peek_DoesNotChangeCount`